.travis.yml

```diff
@@ -2,16 +2,31 @@
 # https://stackoverflow.com/questions/27644586/how-to-set-up-travis-ci-with-multiple-languages
 
 matrix:
-  include:
-    - language: go
-      go:
-        - "1.11.x"
-      before_install:
-        - go get golang.org/x/tools/cmd/goimports
-        - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1
+    include:
+        -   language: go
+            go:
+                - "1.11.x"
```

Comment:
This is not in line with your mod file, which is 1.13. Also, for the home dir suggestion I made you need 1.12+